lib/interface.tsx

```diff
@@ -1,14 +1,31 @@
+import { array } from 'yup'
+
 export type HomePageType = {
   data: {
     homepage: Homepage
     common: Common
   }
 }
+export type WhatWeDoCardType = {
+  card: {
```

Comment:
This is nice but if you did it here you should remove it from line 40 and use that type there instead. now you got it duplicated.